pystachio/__init__.py

```diff
@@ -15,6 +15,9 @@
     Float,
     Integer,
     String)
+from .choice import (
+   Choice
```

Comment:
Sorry. Didn't even notice; I'm notoriously  bad at seeing whitespace, so I never even noticed that my editor settings weren't right for this project.